src/components/EmptyStateAvatars.js

```diff
@@ -0,0 +1,73 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import {Image, View} from 'react-native';
+import styles from '../styles/styles';
+import Text from './Text';
+import CONST from '../CONST';
+import Avatar from './Avatar';
+
+const propTypes = {
+    /** Array of avatar URL */
+    avatarImageURLs: PropTypes.arrayOf(PropTypes.string),
+
+    /** Whether this avatar is for an custom room */
+    isCustomChatRoom: PropTypes.bool,
+};
+
+const defaultProps = {
+    avatarImageURLs: [],
+    isCustomChatRoom: false,
+};
+
+const EmptyStateAvatars = ({avatarImageURLs, isCustomChatRoom}) => {
```

Comment:
I have thoughts about how to clean up a lot of our Avatar components and make them more reusable, but most of those are out of scope for now. In the meantime, let's rename this `isCustomChatRoom` prop to `isDefaultChatRoom` to match the prop name in other components.

Reply:
yeah updated